internal/engine/wazevo/backend/isa/arm64/cond.go

```diff
@@ -213,86 +213,3 @@ func condFlagFromSSAFloatCmpCond(c ssa.FloatCmpCond) condFlag {
 		panic(c)
 	}
 }
-
-// vecArrangement is the arrangement of data within a vector register.
```

Comment:
also I moved them from this file (these are nothing to do with "cond" ) FYI @evacchi